test/models/tenant_test.rb

```diff
@@ -13,30 +15,29 @@ class TenantTest < ModelCase
 
   test '#usage_count returns the expected amount when the tenant has already been used' do
     tenant = create :tenant
-    Redis.current.set tenant.id, 10
+    tenant_calls = 10
+
+    tenant_calls.times.each { |_n| tenant.register_usage }
 
-    assert 10 == tenant.usage_count
+    assert tenant.usage_count == tenant_calls
   end
 
   test '#usage_count returns the expected amount when the tenant has never been used' do
     tenant = create :tenant
-    Redis.current.set tenant.id, nil
 
-    assert 0 == tenant.usage_count
+    assert tenant.usage_count.zero?
   end
 
-  test '#register_usage increments the tenant usage by 1'  do
+  test '#register_usage increments the tenant usage by 1' do
     tenant = create :tenant
-    Redis.current.set tenant.id, 0
 
     tenant.register_usage
-    assert 1 == tenant.usage_count
+    assert tenant.usage_count == 1
```

Comment:
Is `assert actual == expected` a pattern? I've been doing the opposite for a long time 😄

Reply:
I honestly don't know it's just the default on rubocop (this change was automated, but on Elixir I do the opposite)